src/components/folder/ContantList.tsx

```diff
@@ -0,0 +1,27 @@
+import PostCard from './PostCard';
+import { EmptyBox, PostCardWrap } from '../../pages/folder/folderStyle';
+import { IFolderContent } from '../../pages/folder/interface';
+
+interface IFolderList {
+  contant: IFolderContent[] | [] | undefined | null;
+  loading: boolean;
+}
+
+function ContantList({ contant, loading }: IFolderList) {
+  return (
+    <div>
+      {loading ? (
+        <EmptyBox>Loading...</EmptyBox>
+      ) : contant ? (
+        <PostCardWrap>
+          {contant?.map((data) => (
+            <PostCard key={data.id} {...data} />
+          ))}
+        </PostCardWrap>
+      ) : (
+        <EmptyBox>저장된 링크가 없습니다.</EmptyBox>
+      )}
```

Comment:
렌더부에 3항 연산자를 중복으로 쓰면 가독성이 많이 떨어지게 되어서 loading 일 때를 따로 early return 해주시는 걸 추천드려요!
```suggestion
if(loading) {
   return <EmptyBox>Loading...</EmptyBox>
}

return 
      contant ? (
        <PostCardWrap>
          {contant?.map((data) => (
            <PostCard key={data.id} {...data} />
          ))}
        </PostCardWrap>
      ) : (
        <EmptyBox>저장된 링크가 없습니다.</EmptyBox>
      )}
```